routes/html-content.js

```diff
@@ -0,0 +1,7 @@
+/**
+* Return true if the request ask for html content, false otherwise
+*/
+module.exports = function (request) {
+  var htmlContent = request.headers.accept && request.headers.accept.indexOf('text/html') > -1;
+  return Boolean(htmlContent);
```

Comment:
We should probably return HTML as the default, check for a JSON accept header and return JSON then. I know the twitter crawler bot doesn't send a header, but expects HTML as the response